app/scripts/views/sign_up.js

```diff
@@ -218,6 +222,15 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       return this._isMailcheckEnabledValue;
     },
 
+    passwordCheck: function () {
+      var self = this;
+      var password = this.getElementValue('.password');
+      this.passwordcheck(password, function (msg) {
+        self.logScreenEvent(msg);
```

Comment:
I forgot to add to the last comment, you are going to have to add new errors to auth-errors.js.